### PR TITLE
[de-CH] add HassClimateGetTemperature and smaller changes

### PR DESCRIPTION
**Pull request by dontinelli**
I added first sentences for HassClimateGetTemperature including test.

Furthermore I corrected the "excludes_context"-syntax which enabled to add test-sentences for turning on/off only one light.

Finally I tidied up the code a bit, including adjustments of formatting to the best practise (as I understood) and adding options for d' with and without whitespace.

Locally all tests passed.

### CODE REVIEW

**Review by schizza — APPROVED** [review]
I have fixed `exclude_context` in `homeassistant_` domain.

File: sentences/de-CH/_common.yaml [hunk 1]
```diff
@@ -49,7 +51,7 @@ lists:
       - in: "(f|fahrenheit)"
         out: "Fahrenheit"
 expansion_rules:
-  name: "[(d'|d|de|dr|der|die|ds|s)] {name}"
+  name: "([(d'|d|de|dr|der|die|ds|s)] {name}|d'{name})"
```
Comment (schizza) [inline sentences/de-CH/_common.yaml:54]: You can define `name` as this: `"[(d'[' ']|d |de |dr |der |die |ds |s )]{name}"`

File: sentences/de-CH/_common.yaml [hunk 2]
```diff
@@ -58,9 +60,13 @@ expansion_rules:
   temp: "{temperature} [Grad] [{temperature_unit}]"
   schliessen: "(tue {name} (zue|abe)|mach {name} (zue|abe)|schliess {name})"
   mach: "(mach|tue)"
-  Liechter: "([(d'|d|alli|aui)] Liechter|[(d'|d|alli|aui)] Lampä|[(d'|d|alli|aui)] Lampe|[(d'|d|alli|aui)] Lampäne)"
-  Liecht: "([(ds|s)] Liecht|[d] Lampe|[d] Lampä)"
-  Geräte: "([(d'|d|alli|aui)] (Grät|Entitäte|Entitätä))"
+  Liechter: "([(d'|d' |d |alli |aui )]Liechter|[(d'|d' |d |alli |aui )]Lampä|[(d'|d' |d |alli |aui )]Lampe|[(d'|d' |d |alli |aui )]Lampäne)"
```
Comment (schizza) [inline sentences/de-CH/_common.yaml:63]: Also here:
`Liechter: "([(d'[' ']|d |alli |aui )]Liechter|[(d'[' ']|d |alli |aui )]Lampä|[(d'[' ']|d |alli |aui )]Lampe|[(d'[' ']|d |alli |aui )]Lampäne)"`

File: sentences/de-CH/_common.yaml (hunk 1 above)
Comment (dontinelli) [reply sentences/de-CH/_common.yaml]: I see. Is there any rule what should be used? While your version is shorter,  it seems more difficult to read (for humans) due to the additional [].

Comment (schizza) [reply sentences/de-CH/_common.yaml]: Nope, there is no rule for this.

**Review by schizza — APPROVED** [review]

**Comment by schizza** [comment]
@dontinelli please resolve conflicts, as main branch has changed and I am unable to resolve this as I do not know which one is grammatically correct.

**Pull request merged**